template_example/dev/ansible/03_healthcheck_test.yml

```
@@ -0,0 +1,34 @@
+- name: set variable
+  set_fact:
+    consul_token: "master"
```

```suggestion
    consul_token: "{{ consul_master_token }}"
```
Can also consider removing the fact and put `"{{ consul_master_token }}"` in line `25`

---

Access might be done via lookup. Here is [example](https://github.com/fredrikhgrelland/vagrant-hashistack/blob/master/ansible/files/bootstrap/consul/post/task/vault-integration.yml#L5)

---

Yes! My bad forgot to add lookup - will edit the orig post

---

Thanks for your comments @zhenik @claesgill 
Setting the variable explicitly at the beginning of the file, has been a conscious choice as the **consul_master_token** is not available to use as we are not using the 0.6.0 version of the box. This variable is available only in 0.6.0 version of the box.https://github.com/fredrikhgrelland/vagrant-hashistack-template/blob/7ad8b6a36d29b5da71c179493e553e6f9b4a6fa7/Vagrantfile.default#L3

And, for 0.5.0 version of vagrant box we took away the default variables [Remove default tokens #373](https://github.com/fredrikhgrelland/vagrant-hashistack/pull/373/files?file-filters%5B%5D=.j2&file-filters%5B%5D=.tf&file-filters%5B%5D=.yml&file-filters%5B%5D=No+extension#diff-fd33301bf1a987ab2ba1449a2c970b1cbe4cc34a50a454ef6a4bae0ae3e96111L14) from the .env file so basically, we do not have the consul_token available in the environment to look up for.

---

Oh, then we should probably update the box version as well?

---

Yeah sure...but that is out of the scope for this PR ;) 
Also, I guess that would mean additional task of synching the terraform-modules and fixing them as per the changes done to 0.6.0 box.

---

That's right. Should stick to the related issue :)